conans/test/functional/generators/virtualenv_test.py

```diff
@@ -156,6 +156,7 @@ def _run_virtualenv(self, generator):
         with environment_append({"PATH": [self.ori_path, ]}):
             # FIXME: I need this context because restore values for the 'deactivate' script are
             #        generated at the 'generator.content' and not when the 'activate' is called.
```

Comment:
This is fixed in this PR

Reply:
Could we remove the context then?

Reply:
Indeed. And soon we will be able to apply nested ones